examples/container-service-extension-3.1.x/3.7-cse-3.1.x-install.tf

```diff
@@ -25,12 +26,142 @@ terraform {
   }
 }
 
+# We'll use the System Administrator only to create the service role, the service user to manage CSE and the minimum
+# objects required (Organization, VDC, ALB, etc). The remaining items will be created by this service user.
+
 provider "vcd" {
```

Comment:
It may be only me, but I'd like to see both providers next to each other, as otherwise it's easy to miss that there are two. Maybe we should move this vcd.administrator provider right below the default one?

Reply:
Done, I've re-ordered it a bit + some changed explanations.